enhancements/api-review/custom-tags-aws.md

```diff
@@ -0,0 +1,144 @@
+---
+title: apply-user-defined-tags-to-all-aws-resources-created-by-openshift
+authors:
+  - "@gregsheremeta"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-03-24
+last-updated: 2021-03-24
+status: implementable
+---
+
+# Apply user defined tags to all AWS resources created by OpenShift
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement describes a proposal to allow an administrator of OpenShift to
+have the ability to apply user defined tags to all resources created by OpenShift in AWS.
+
+Note: this enhancement is slightly retroactive. Work has already begun on this. See
+ - https://github.com/openshift/api/pull/864
+ - https://github.com/openshift/cluster-ingress-operator/pull/578
+
+## Motivation
+
+Motivations include but are not limited to:
+
+ - allow admin, compliance, and security teams to keep track of assets and objects created by OpenShift,
+   both at install time during continuous operation (Day 2)
+ - in a Managed OpenShift environment such as Red Hat OpenShift on AWS (ROSA), allow easy differentiation
+   between Red Hat-managed AWS objects and customer-managed objects
+ - *allow for the restriction of permissions granted to Red Hat in an AWS account by AWS resource tags.
+   For example, see SDE-1146 - "IAM users and roles can only operate on resources with specific tags"*
```

Comment:
Is the SDE Jira public, maybe put a link here? To make "see SDE-1146" easier

Reply:
done